package.json

```diff
@@ -13,7 +13,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start-backend": "nodemon ./src/backend/App.js localhost 3001"
```

Comment:
I believe you don't really need `localhost 3001` here

Reply:
Ok, So I just need to delete and that set?

Reply:
Yep, the line should look like:

```
"start-backend": "nodemon ./src/backend/App.js"
```